tests/test_api.py

```diff
@@ -337,12 +342,15 @@ async def test_patch_access_request(
     # check that an event was published for 'access request allowed'
     assert len(recorder.recorded_events) == 1
     recorded_event = recorder.recorded_events[0]
-    assert recorded_event.key == CREATION_DATA["user_id"]
-    assert recorded_event.payload == {
-        "user_id": CREATION_DATA["user_id"],
-        "dataset_id": CREATION_DATA["dataset_id"],
-    }
-    assert recorded_event.type_ == rest.config.access_request_allowed_type
+    assert recorded_event.key == access_request_id
+    for key in [
+        "user_id",
+        "dataset_id",
+        "dataset_title",
+        "dac_alias",
```

Comment:
You should also check `request_text`, `access_ends` and that the status in the payload is pending.

Reply:
Similarly above